libs/gi-art-scanner/src/lib/processImg.ts

```diff
@@ -369,3 +371,29 @@ function parseRarity(
   }
   return clamp(count, 1, 5)
 }
+
+function findTitle(artifactCardImageData: ImageData) {
+  const width = artifactCardImageData.width
+  const widthThreshold = width * 0.7
+
+  function findTitleColored(darkerTitleColor: Color, LighterTitleColor: Color) {
+    const hist = histogramContAnalysis(
+      artifactCardImageData,
+      darkerColor(darkerTitleColor, 20),
+      lighterColor(LighterTitleColor, 20),
+      false,
+      [0, 0.3] // only scan the top 30% of the img
+    )
+    if (hist.find((v) => v > widthThreshold)) return hist
+    return null
+  }
+  const titleColors = [
+    [goldenTitleDarkerColor, goldenTitleLighterColor],
+    [purpleTitleDarkerColor, purpleTitleLighterColor],
+    [blueTitleDarkerColor, blueTitleLighterColor],
+  ] as const
+  return titleColors.reduce(
```

Comment:
Can you add a comment please?
```suggestion
  // Return first detected title color
  return titleColors.reduce(
```